app.rb

```diff
@@ -0,0 +1,123 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @classroom = Classroom.new('Grade 10')
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'No book added, please add a new book'
+      return
+    end
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'No people registered, please add new people'
+      return
+    end
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name} ID: #{person.id} Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a teacher or student? Enter 1 for the student and 2 for teacher.'
+    option = gets.chomp
+
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid! Please enter 1 or 2'
+    end
+  end
+
+  def create_student
+    print 'Name:'
+    name = gets.chomp
+
+    print 'Age:'
+    age = gets.chomp.to_i
+
+    print 'Has parent permission? [Y/N]'
+    parent_permission = gets.chomp.downcase
+
+    student = Student.new(age, @classroom, name, parent_permission: parent_permission == 'y')
```

Comment:
- [ ] You did a great job fixing the previous issue. :+1: Let's work on another issue to achieve the objective. When creating a student the age is assigned to the name and vice versa. Everything is working fine when creating a teacher. Please check the attached screenshot as a reference. ✍️

![image](https://user-images.githubusercontent.com/17243325/235285358-4a23095e-b5e1-46ee-a427-7e333bb1f042.png)